test/integration/targets/flatpak_remote/aliases

```diff
@@ -0,0 +1,7 @@
+unsupported
```

Comment:
@mattclay does this aliases list seem right to you?

Reply:
Please see my comment on PR #42169

Reply:
@webknjaz Since you merged #42169, could you please merge this one as well?

Reply:
Done!